module.exports = {
  'ChEA': 'ChEA contains results from transcription factor ChIP-seq studies extracted from supporting materials of publications. Each entry has the transcription factor, PMID, cell type and organism. A peak at the promoter of gene {0} was detected in each study.',
  'TRANSFAC_and_JASPAR_PWMs': 'Using the PWM from TRANSFAC and JASPAR the binding motifs were detected at the promoter of gene {0}.',
  'Genome_Browser_PWMs': 'Using a published list from the UCSC Genome Browser gene {0} was associated with these binding motifs.',
  'Epigenomics_Roadmap_HM_ChIP-seq': 'Processed ChIP-seq data from the Epigenomics Roadmap project to associate detected peaks near genes.',
  'TargetScan_microRNA': 'Computationally predicted targets for microRNAs.',
  'ENCODE_TF_ChIP-seq_2015': 'Processed ChIP-seq data from the ENCODE project to associate detected peaks near genes.',
  'TF-LOF_Expression_from_GEO': 'Gene expression signatures extracted from the GEO database for transcription factor perturbations.',
  'ENCODE_Histone_Modifications_2015': 'Processed ChIP-seq data from the ENCODE project to associate detected peaks near genes.',
  'Transcription_Factor_PPIs': 'Protein-protein interactions for transcription factors extracted from several literature-based databases.',
  'KEGG_2015': 'KEGG\'s cell signaling pathway database.',
  'WikiPathways_2015': 'WikiPathways\'s cell signaling pathway database.',
  'Reactome_2015': 'Reactome\'s cell signaling pathway database.',
  'BioCarta_2015': 'Metabolic pathways from BioCarta.',
  'PPI_Hub_Proteins': 'Protein-protein interactions for proteins with more than 50 known partners extracted from several literature-based databases.',
  'KEA_2015': 'Kinase enrichment analysis (KEA) database of kinase-substrate interactions extracted from literature.',
  'NURSA_Human_Endogenous_Complexome': 'Protein complexes identified by IP-MS mass spec by the NURSA project.',
  'CORUM': 'Protein complexes identified by mass spec from the CORUM database.',
  'SILAC_Phosphoproteomics': 'Up and down differentially phosphorylated phosphosites extracted from SILAC phosphoproteomics experiments.',
  'Kinase_Perturbations_from_GEO': 'Gene expression signatures extracted from the GEO database for kinase gene perturbations.',
  'Kinase_Perturbations_from_L1000': 'Metabolic pathways from Kinase_Perturbations_from_L1000.',
  'Phosphatase_Substrates_from_DEPOD': 'Phosphatase-substrate interactions from the DEPOD database.',
  'HumanCyc': 'Metabolic pathways from HumanCyc.',
  'NCI-Nature': 'Cell signaling pathways from NCI-Nature pathway database.',
  'Panther': 'Metabolic and cell signaling pathways from the Panther database.',
  'GO_Biological_Process': 'Gene Ontology Biological Processes where the ontology tree was cut at an intermediate level to generate gene sets',
  'GO_Cellular_Component': 'Gene Ontology Cellular Components where the ontology tree was cut at an intermediate level to generate gene sets',
  'GO_Molecular_Function': 'Gene Ontology Molecular Functions where the ontology tree was cut at an intermediate level to generate gene sets',
  'MGI_Mammalian_Phenotype_Level_3': 'Knockout mouse phenotypes organized into an ontology. ',
  'MGI_Mammalian_Phenotype_Level_4': 'Knockout mouse phenotypes organized into an ontology that is cut at level 3 of the. ',
  'Human_Phenotype_Ontology': 'HPO is an effort to organize human phenotypes into an ontology.',
  'CMAP_up': 'The top 500 up-regulated genes from the Old Connectivity map gene rank table.',
  'CMAP_down': 'The top 500 down-regulated genes from the Old Connectivity map gene rank table.',
  'GeneSigDB': 'The entire GeneSigDB database from the Broad.',
  'OMIM_Disease': 'The OMIM database listing gene associated with human diseases. ',
  'OMIM_Expanded': 'Protein-protein interaction networks that connect disease genes from OMIM.',
  'VirusMINT': 'Human genes that are known to physically interact with viral genes.',
  'MSigDB_Computational': 'Modules of genes mined by processing cancer microarray data.',
  'MSigDB_Oncogenic_Signatures': 'Cell signaling in cancer reconstructed from expression data.',
  'Virus_Perturbations_from_GEO_up': 'Differentially expression genes before and after viral infection of mammalian cells.',
  'Virus_Perturbations_from_GEO_down': 'Differentially expression genes before and after viral infection of mammalian cells.',
  'Achilles_fitness_increase': 'Knockdown genes that increased cell viability of cancer cell lines from the Achillis project.',
  'Achilles_fitness_decrease': 'Knockdown genes that decreased cell viability of cancer cell lines from the Achillis project.',
  'Human_Gene_Atlas': 'Up regulated genes in human tissues from BioGPS.',
  'Mouse_Gene_Atlas': 'Up regulated genes in mouse tissues from BioGPS.',
  'Cancer_Cell_Line_Encyclopedia': 'Up regulated genes across many cancer cell lines from CCLE.',
  'NCI-60_Cancer_Cell_Lines': 'Up regulated genes across the NCI-60 panel of cancer cell lines.',
  'Tissue_Protein_Expression_from_ProteomicsDB': 'Up regulated proteins in various human tissues.',
  'Tissue_Protein_Expression_from_Human_Proteome_Map': 'Up regulated proteins in various human tissues from ProteomicsDB.',
  'Allen_Brain_Atlas_up': 'Down regulated genes across brain regions profiled by the Allen Brain Atlas.',
  'Allen_Brain_Atlas_down': 'Up regulated genes across brain regions profiled by the Allen Brain Atlas.',
  'ESCAPE': 'Gene sets collected for the ESCAPE stem cell focused database.',
  'Chromosome_Location': 'Chromosome segment location of human genes.',
  'HMDB_Metabolites': 'Metabolites that interact with human genes.',
  'Pfam_InterPro_Domains': 'Structural protein domains that are shared among proteins.',
  'HomoloGene': 'Gene sets created based on their evolutionary age.',
  'Genes_Associated_with_NIH_Grants': 'Gene set created by converting publications associated with NIH grants to genes using GeneRIF.',
  'KEGG_2013': 'Legacy version of the KEGG pathways gene set library.',
  'WikiPathways_2013': 'Legacy version of the WikiPathways gene set library.',
  'Reactome_2013': 'Legacy version of the Reactome gene set library.',
  'BioCarta_2013': 'Legacy version of the Biocarta pathways gene set library.',
  'ENCODE_TF_ChIP-seq_2014': 'Legacy version of the ECONDE transcription factors gene set library.',
  'KEA_2013': 'Legacy version of the Kinase Enrichment Analysis database gene set library.',
  'GO_Biological_Process_2013': 'Legacy version of the GO biological process gene set library.',
  'GO_Cellular_Component_2013': 'Legacy version of the GO cellular component gene set library.',
  'Drug_Perturbations_from_GEO_2014': 'Legacy version of the drug perturbation extracted from GEO gene set library.',
  'GO_Molecular_Function_2013': 'Legacy version of the GO molecular functions gene set library.',
  'Disease_Signatures_from_GEO_up_2014': 'Legacy version of the disease sigantures extracted from GEO down genes gene set library.',
  'Disease_Signatures_from_GEO_down_2014': 'Legacy version of the disease sigantures extracted from GEO up genes gene set library.',
  'ENCODE_Histone_Modifications_2013': 'Legacy version of the ENCODE histome modifications gene set library.',
  'MGI_Mammalian_Phenotype_2013': 'Legacy version of the MGI mammalian phenotypes gene set library.',
  'Disease_Perturbations_from_GEO_down': 'Gene sets extarcted from GEO comparing normal vs. diseased tissues.',
  'Disease_Perturbations_from_GEO_up': 'Gene sets extarcted from GEO comparing normal vs. diseased tissues.',
  'Drug_Perturbations_from_GEO_down': 'Gene sets extarcted from GEO comparing cells before and after drug treatment.',
  'Drug_Perturbations_from_GEO_up': 'Gene sets extarcted from GEO comparing cells before and after drug treatment.',
  'Single_Gene_Perturbations_from_GEO_up': 'Gene sets extarcted from GEO comparing cells before and after genetic manipulation of a single gene.',
  'Single_Gene_Perturbations_from_GEO_down': 'Gene sets extarcted from GEO comparing cells before and after genetic manipulation of a single gene.',
};
